src/test/java/codeu/controller/ProfilePageServletTest.java

```diff
@@ -0,0 +1,42 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+
+public class ProfilePageServletTest {
+
+    private ProfilePageServlet profilePageServlet;
+    private UserStore mockUserStore;
+    private HttpServletRequest mockRequest;
+    private HttpServletResponse mockResponse;
+    private RequestDispatcher mockRequestDispatcher;
+    private ConversationStore mockConversationStore;
+
+
+    @Before
+    public void setup() {
+        profilePageServlet = new ProfilePageServlet();
+        mockRequest = Mockito.mock(HttpServletRequest.class);
+        mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/profile.jsp"))
+                .thenReturn(mockRequestDispatcher);
+        mockResponse = Mockito.mock(HttpServletResponse.class);
+    }
+
+    @Test
+    public void testDoGet() throws IOException, ServletException {
```

Comment:
You could add this test function? It's up to you- if you want to learn more about testing this is a good way to practice, but I'm not going to be too strict about it.